aws-rds-dbclusterendpoint/src/main/java/software/amazon/rds/dbclusterendpoint/UpdateHandler.java

```diff
@@ -0,0 +1,75 @@
+package software.amazon.rds.dbclusterendpoint;
+
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.rds.common.handler.Commons;
+import software.amazon.rds.common.handler.HandlerConfig;
+import software.amazon.rds.common.handler.Tagging;
+
+
+public class UpdateHandler extends BaseHandlerStd {
+
+
+    public UpdateHandler() {
+        this(HandlerConfig.builder()
+                .backoff(BACKOFF_DELAY)
+                .build());
+    }
+
+    public UpdateHandler(HandlerConfig config) {
+        super(config);
+    }
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient,
+            final Logger logger
+    ) {
+        final ResourceModel desiredModel = request.getDesiredResourceState();
+
+        final Tagging.TagSet previousTags = Tagging.TagSet.builder()
+                .systemTags(Tagging.translateTagsToSdk(request.getPreviousSystemTags()))
+                .stackTags(Tagging.translateTagsToSdk(request.getPreviousResourceTags()))
+                .resourceTags(Translator.translateTagsToSdk(request.getPreviousResourceState().getTags()))
+                .build();
+
+        final Tagging.TagSet desiredTags = Tagging.TagSet.builder()
+                .systemTags(Tagging.translateTagsToSdk(request.getSystemTags()))
+                .stackTags(Tagging.translateTagsToSdk(request.getDesiredResourceTags()))
+                .resourceTags(Translator.translateTagsToSdk(request.getDesiredResourceState().getTags()))
+                .build();
+
+        return ProgressEvent.progress(desiredModel, callbackContext)
+                .then(progress -> updateEndpoint(proxy, request, callbackContext, proxyClient))
+                .then(progress -> updateTags(proxy, proxyClient, progress, previousTags, desiredTags))
+                .then(progress -> new ReadHandler().handleRequest(proxy, request, callbackContext, proxyClient, logger));
+    }
+
+    protected ProgressEvent<ResourceModel, CallbackContext> updateEndpoint(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient
+    ) {
+        return proxy.initiate("rds::update-cluster-endpoint", proxyClient, request.getDesiredResourceState(), callbackContext)
```

Comment:
Same as create, Modify can throw `InvalidDBInstanceState` or `InvalidDBClusterStateFault` which need to back off and retry after some time.